simulariumio/trajectory_converter.py

```diff
@@ -56,374 +53,6 @@ def __init__(self, input_data: TrajectoryData):
         """
         self._data = input_data
 
```

Comment:
moved these functions to the writers, split up between JsonWriter and base Writer (for functions that are reused by BinaryWriter)